src/test/java/net/openhft/chronicle/wire/JSONWireTest.java

```diff
@@ -256,6 +288,7 @@ public void testMarshallableWithTwoLists() {
                     "}\n", lists1.toString());
             final String str = JSON.asString(lists1);
             testCopyToBinaryAndBack(str);
+//            testCopyToYAMLAndBack(str);
```

Comment:
Did you mean to leave this commented-out?

Reply:
Hi @batespj , thanks for spotting - I have backported Peter's commit, cherry-picking it exactly as appears on develop. So I think its best we leave as is